src/components/authentication/SignUpFields.tsx

```diff
@@ -27,8 +44,43 @@ function SignUpFields(props: Props) {
         id="password-confirm-input"
         label="Confirm password"
         placeholder="Confirmation"
+        helperText={confirmPasswordHelperText}
+        error={!!confirmPasswordHelperText}
         onChange={(e: any) => onChangeData("confirmPassword", e)}
       />
+      <Typography className={styles.signupFields__TextSeparator} variant="h6">
+        Personal Details
+      </Typography>
+      <RadioGroup row aria-label="position" name="position" defaultValue="">
+        <FormControlLabel
+          value="Ms."
+          control={<Radio color="primary" />}
+          label="Ms."
+          labelPlacement="start"
+          onChange={(e: any) => onChangeData("civility", e)}
+        />
+        <FormControlLabel
+          value="Mr."
+          control={<Radio color="primary" />}
+          label="Mr."
+          labelPlacement="start"
+          onChange={(e: any) => onChangeData("civility", e)}
+        />
+        <FormControlLabel
+          value="Dr."
+          control={<Radio color="primary" />}
+          label="Dr."
+          labelPlacement="start"
+          onChange={(e: any) => onChangeData("civility", e)}
+        />
+        <FormControlLabel
+          value={""}
+          control={<Radio color="primary" />}
+          label="Leave blank"
+          labelPlacement="start"
+          onChange={(e: any) => onChangeData("civility", e)}
+        />
```

Comment:
define a list with the 4 values of civility and map on it to return <FormControlLabel /> --> remove code duplication :)